File: URI/Python/Uri 1009 - Salário com Bônus.py
nome_funci = str(input('Nome do funcionário: ', ))
salario = float(input('Salário: ', ))
format(s,'.2f')
vendas = float(input('Vendas realizadas: ', ))
format(v,'.2f')
bonus = 0.15

total = (bonus*vendas) + salario

print("Para o funcionário",nome_funci,'o salário toral é de R$:', format(t,'.2f'))



#Para o URI:


n = str(input())
s = float(input())
format(s,'.2f')
v = float(input())
format(v,'.2f')
print("TOTAL = R$", format(((v*0.15)+s),'.2f'))

File: URI/C++/Uri 1009 - Salário com Bônus.cpp
#include <iostream>                                     // Biblioteca padrão entrada/saída
#include <iomanip>                                      // Biblioteca para definir o número de casas decimais
#include <string>                                       // Biblioteca para variável do tipo "string"s

using namespace std;

int main() {                                            // Função MAIN:
    double sal, vendas;                                     // Variáveis para entrada
    string nome;

    cin >> nome;                                            // Entradas
    cin >> sal;
    cin >> vendas;

    cout << fixed << setprecision(2);                       // Define a saída com duas casas decimais
    cout << "TOTAL = R$ " << sal + vendas*0.15 << endl;     // Mostra no termial

    // system("pause");
	return 0;
}